actor/v7action/service_instance.go

```diff
@@ -87,13 +91,12 @@ func (actor Actor) UpdateManagedServiceInstance(serviceInstanceName, spaceGUID s
 
 	switch err.(type) {
 	case nil:
+		return Warnings(warnings), nil
```

Comment:
Theoretically this case is not needed (the default would cover it) but might be a good idea for make the behaviour explicit anyways